src/main/java/mate/academy/spring/controller/UserController.java

```diff
@@ -0,0 +1,26 @@
+package mate.academy.spring.controller;
+
+import mate.academy.spring.mapper.impl.response.UserResponseMapper;
+import mate.academy.spring.model.dto.response.UserResponseDto;
+import mate.academy.spring.service.UserService;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+@RequestMapping("/users")
+public class UserController {
+    private final UserService userService;
+    private final UserResponseMapper responseMapper;
```

Comment:
it is better to use abstraction instead of implementation here and in other controllers
https://mate-academy.github.io/jv-program-common-mistakes/java-spring/rest-practice/jv-spring-rest-practice_checklist#:~:text=Use%20DtoRequestMapper%20and%20DtoResponseMapper%20as%20types%20when%20you%20declare%20mapper%20fields%20in%20controllers.